containers/gcc14-ubsan/Dockerfile

```diff
@@ -0,0 +1,143 @@
+FROM fedora:40
+
+# ------------------------------------------------------------------------------------
+# Install pre-built R and requirements
+
+RUN dnf -y upgrade && \
+    dnf -y install dnf-plugins-core && \
+    dnf -y install curl && \
+    curl -LO "https://github.com/r-hub/containers/releases/download/latest/R-rstudio-fedora-40-devel-gcc14-1-1.$(arch).rpm" && \
+    yum install -y R-*.rpm && \
+    rm R-*.rpm
+
+# Extra system packages that pak does not know about
+RUN dnf install -y flexiblas
+RUN dnf install -y https://rpms.r-pkg.org/fedora-38/jags-4.3.2-1.$(arch).rpm
+
+# ------------------------------------------------------------------------------------
+# Put R on PATH
+# Auto-install system requirements
+
+ENV PATH="/opt/R/devel-gcc14/bin:${PATH}"
+ENV PKG_SYSREQS=true
+ENV R_PKG_SYSREQS2=true
+
+# ------------------------------------------------------------------------------------
+# Set CRAN repo, no P3M for Fedora
+
+RUN echo 'options(repos =c(CRAN = "https://cran.rstudio.com"))' \
+    >> /opt/R/devel-gcc14/lib/R/library/base/R/Rprofile
+
+RUN if [ "$(arch)" = "x86_64" ]; then \
+        . /etc/os-release; \
+        RVER=$(R -s -e 'cat(paste(getRversion()[,1:2]))'); \
+        echo "options(repos = c(RHUB = 'https://raw.githubusercontent.com/r-hub/repos/main/${ID}-${VERSION_ID}/${RVER}', getOption('repos')))" \
+            >> /opt/R/devel-gcc14/lib/R/library/base/R/Rprofile; \
+        echo 'options(HTTPUserAgent = sprintf("R/%s R (%s)", getRversion(), paste(getRversion(), R.version["platform"], R.version["arch"], R.version["os"])))' \
+            >> /opt/R/devel-gcc14/lib/R/library/base/R/Rprofile; \
+    fi
+
+# ------------------------------------------------------------------------------------
+# Install pak
+
+RUN /opt/R/devel-gcc14/bin/R -q -e \
+    'install.packages("pak", repos = sprintf("https://r-lib.github.io/p/pak/%s/%s/%s/%s", "devel", .Platform$pkgType, R.Version()$os, R.Version()$arch))'
+
+# ------------------------------------------------------------------------------------
+# Install pkgbuild, rcmdcheck, and decor
```

Comment:
Why is this needed? It is also error prone, because `install.packages()` does not error on installation failures.

Reply:
not really needed
the pak part was copied from the gcc14 dockerfile